src/Components/ADT3DSceneBuilder/Internal/Behaviors/Widgets/WidgetBuilders/DataHistoryWidgetBuilder/Internal/TimeSeriesFormCallout.tsx

```diff
@@ -152,9 +161,21 @@ const TimeSeriesFormCallout: React.FC<IProp> = ({
                         expression: seriesToEdit?.expression
                     }}
                     onChange={handlePropertyChange}
-                    allowedPropertyValueTypes={numericPropertyValueTypes}
+                    allowedPropertyValueTypes={
+                        allowedTimeseriesPropertyValueTypes
+                    }
                     required
                 />
+                {seriesToEdit?.propertyType &&
```

Comment:
can you please include a screenshot in the PR description of what this looks like with and without this message being shown?

Reply:
added